edgelet/edgelet-docker/src/runtime.rs

```diff
@@ -1122,8 +1122,11 @@ impl Authenticator for DockerModuleRuntime {
 
 fn init_client(docker_url: &Url) -> Result<DockerClient<UrlConnector>> {
     // build the hyper client
-    let client =
-        Client::builder().build(UrlConnector::new(docker_url).context(ErrorKind::Initialization)?);
+    let client = Client::builder()
+        // we don't need connection pool'ing for local docker socket.
+        .keep_alive(false)
```

Comment:
Do you need to set keep_alive here as well, then? https://github.com/Azure/iot-identity-service/blob/main/http-common/src/connector.rs#L242

Reply:
There's no keep-alive in new hyper, so it's only a 1.2 problem.

Reply:
Correct ^

Reply:
Got it.. thanks..